src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,7 +24,7 @@
 
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
-    private final FilteredList<Person> filteredPersons;
+    private FilteredList<Person> filteredPersons;
```

Comment:
see sortFilteredPersonList() for comments

```suggestion
    private final SortedList<Person> sortedPersons;
    private final FilteredList<Person> filteredPersons;
```

Reply:
In ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) constructor
Change from this
`filteredPersons = new FilteredList<>(this.addressBook.getPersonList());`

to this
`
sortedPersons = new SortedList<>(this.addressBook.getPersonList());
`
`
filteredPersons = new FilteredList<>(sortedPersons);
`

Reply:
Thanks Chris for the suggestion to wrap filteredList around a SortedList! I've updated the ModelManager to reflect this and it works now 👍